feature-libs/cart/quick-order/components/quick-order/quick-order.component.html

```diff
@@ -1,136 +1,165 @@
+      class="quick-order-deletions-message"
+    >
+    </cx-message>
+  </ng-container>
 
-<ng-container *ngIf="errors$ | async as errors">
-  <cx-message
-    *ngIf="errors.length"
-    (closeMessage)="clearErrors()"
-    [accordionText]="'quickOrderList.errors.reviewErrors' | cxTranslate"
-    [text]="'quickOrderList.errorProceedingToCart' | cxTranslate"
-    [type]="globalMessageType.MSG_TYPE_ERROR"
-    class="quick-order-errors-message"
-  >
-    <ul class="quick-order-errors">
-      <li *ngFor="let error of errors" class="quick-order-error-item">
-        <span>
-          {{
-            'quickOrderList.errors.productIsOutOfStock'
-              | cxTranslate
-                : {
-                    name: error.entry.product.name,
-                    code: error.entry.product.code
-                  }
-          }}
-        </span>
-      </li>
-    </ul>
-  </cx-message>
-</ng-container>
+  <ng-container *ngIf="errors$ | async as errors">
+    <cx-message
+      *ngIf="errors.length"
+      (closeMessage)="clearErrors()"
+      [accordionText]="'quickOrderList.errors.reviewErrors' | cxTranslate"
+      [text]="'quickOrderList.errorProceedingToCart' | cxTranslate"
+      [type]="globalMessageType.MSG_TYPE_ERROR"
+      class="quick-order-errors-message"
+    >
+      <ul class="quick-order-errors">
+        <li *ngFor="let error of errors" class="quick-order-error-item">
+          <span>
+            {{
+              'quickOrderList.errors.productIsOutOfStock'
+                | cxTranslate
+                  : {
+                      name: error.entry.product.name,
+                      code: error.entry.product.code
+                    }
+            }}
+          </span>
+        </li>
+      </ul>
+      <div aria-live="polite" class="cx-visually-hidden">
+        {{
+          'quickOrderList.errors.outOfStockErrorFound'
+            | cxTranslate: { count: errors.length }
+        }}
+      </div>
+    </cx-message>
+  </ng-container>
+
+  <ng-container *ngIf="warnings$ | async as warnings">
+    <cx-message
+      *ngIf="warnings.length"
+      (closeMessage)="clearWarnings()"
+      [accordionText]="'quickOrderList.warnings.reviewWarnings' | cxTranslate"
+      [text]="'quickOrderList.warningProceedingToCart' | cxTranslate"
+      [type]="globalMessageType.MSG_TYPE_WARNING"
+      class="quick-order-warnings-message"
+    >
+      <ul class="quick-order-warnings">
+        <li *ngFor="let warning of warnings" class="quick-order-warning-item">
+          <span>
+            {{
+              'quickOrderList.warnings.productWasReduced'
+                | cxTranslate
+                  : {
+                      name: warning.entry.product.name,
+                      code: warning.entry.product.code,
+                      quantityAdded: warning.quantityAdded
+                    }
+            }}
+          </span>
+        </li>
+      </ul>
+      <div aria-live="polite" class="cx-visually-hidden">
```

Comment:
I think this `aria-live="polite"` attribute is not necessary here, as we have all messages wrapped into another aria-live attribute.